src/ethereum_test_tools/spec/base/base_test.py

```diff
@@ -100,6 +102,7 @@ def fill_info(
         self.info["filling-transition-tool"] = t8n.version()
         if ref_spec is not None:
             ref_spec.write_info(self.info)
+        self.info["framework-version"] = get_framework_version()
```

Comment:
I think we should pass this to the filler as a `str | None`, and ignore if `None`, but the main advantage would be avoid calling `get_framework_version` every fixture we need to fill info for, and reusing the data for all tests.

Reply:
Nice, this could definitely be called only once